tools/sklearn/model_validation.xml

```diff
@@ -64,133 +75,100 @@ y = read_columns(
         c_option = column_option,
         sep='\t',
         header=header,
-        parse_dates=True
-)
-y=y.ravel()
+        parse_dates=True)
+y = y.ravel()
 
+# handle options
 options = params['model_validation_functions']['options']
 splitter, groups = get_cv( options.pop('cv_selector') )
-if groups is None:
-    options['cv'] = splitter
-elif groups == '':
-    options['cv'] = list( splitter.split(X, y, groups=None) )
-else:
-    options['cv'] = list( splitter.split(X, y, groups=groups) )
+options['cv'] = splitter
+options['groups'] = groups
 options['n_jobs'] = N_JOBS
 if 'scoring' in options:
+    primary_scoring = options['scoring']['primary_scoring']
     options['scoring'] = get_scoring(options['scoring'])
 if 'pre_dispatch' in options and options['pre_dispatch'] == '':
     options['pre_dispatch'] = None
 
-pipeline_steps = []
-
-## Set up pre_processor and add to pipeline steps.
-if params['pre_processing']['do_pre_processing'] == 'Yes':
-    preprocessor = params['pre_processing']['pre_processors']['selected_pre_processor']
-    pre_processor_options = params['pre_processing']['pre_processors']['options']
-    my_class = getattr(preprocessing, preprocessor)
-    pipeline_steps.append( ('pre_processor', my_class(**pre_processor_options)) )
-
-## Set up feature selector and add to pipeline steps.
-if params['feature_selection']['do_feature_selection'] == 'Yes':
-    feature_selector = feature_selector(params['feature_selection']['fs_algorithm_selector'])
-    pipeline_steps.append( ('feature_selector', feature_selector) )
-
-## Set up estimator and add to pipeline.
-estimator_json = params['model_validation_functions']['estimator_selector']
-estimator = get_estimator(estimator_json)
+# load pipeline
```

Comment:
I know `##` is recommended comment format for Cheetah. However, in my experience, `#` doesn't harm anything, although it may be tried to be executed.